discord/user.go

```diff
@@ -82,6 +82,7 @@ const (
 	_
 	VerifiedBot
 	VerifiedBotDeveloper
+	CertifiedModerator
```

Comment:
They [changed](https://github.com/discord/discord-api-docs/pull/2946#discussion_r634771408) the name to "Discord Certified Developer", so I think we should do the same.

Reply:
You're right, `Employee` does the same, therefore resolving this.